cloudcix_primitives/bridge_kvm.py

```diff
@@ -0,0 +1,439 @@
+"""
+Primitive for Private VLAN Bridge (KVM only)
+"""
+
+# 3rd party modules
+import jinja2
+# stdlib
+import os
+from typing import Tuple
+# lib
+from cloudcix.rcc import comms_ssh, CHANNEL_SUCCESS
+# local
+from cloudcix_primitives.utils import SSHCommsWrapper, HostErrorFormatter
+
+
+__all__ = [
+    'build',
+    'read',
+    'scrub',
+]
+
+SUCCESS_CODE = 0
+
+template_path = os.path.join(os.path.dirname(__file__), 'templates', __name__.split(".").pop())
+
+def build(
+        host: str,
+        vlan: int,
+        ifname: list,
```

Comment:
```suggestion
        ifname: str,
```

I think based on the comments and where the `ifname` param is called, it's type is a string

Reply:
yeah leftover from a copy paste my bad.